frontend/app/views/spree/shared/_product_tile.html.erb

```diff
@@ -0,0 +1,17 @@
+<% url = spree.product_path(product, taxon_id: @taxon.try(:id)) %>
+<li id="product_<%= product.id %>" class="columns three <%= cycle("alpha", "secondary", "", "omega secondary", name: "classes") %>" data-hook="products_list_item" itemscope itemtype="http://schema.org/Product">
+  <% cache(@taxon.present? ? [I18n.locale, current_pricing_options, @taxon, product] : [I18n.locale, current_pricing_options, product]) do %>
```

Comment:
Is this cache different from the outer one defined when we call the partial? Do you think we could use the same one? 

Maybe it's also worth extracting the cache key somewhere else if they are identical? Having a shared/uniform place where to store how we cache things could be a good idea to explore, probably into another PR.

Reply:
ooh!! good catch, I just moved the code from the other partial without really studying it, will improve it